recipes/khiopsdriver-gcs/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "khiops-driver-gcs" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version.replace("-", "") }}
+
+source:
+  url: https://github.com/KhiopsML/khiopsdriver-gcs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 456daa8f441bf2e461b73949c87307dd7f20ed1e62c4a334a6d4dbcf094cc713
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}                      # [unix]
+    - cmake
+    - ninja
+    - libgoogle-cloud-storage-devel
+    - libboost-devel
+    - libboost-headers
+  host:
+    - fmt
+    - spdlog
+    - libabseil >=20250127.*                 # [win]
```

Please omit this pin so that the global pin is used.
```suggestion
    - libabseil                  # [win]
```